Content.Server/Antag/AntagSelectionSystem.cs

```diff
@@ -298,6 +302,24 @@ public void MakeAntag(Entity<AntagSelectionComponent> ent, ICommonSession? sessi
 
         if (session != null)
         {
+            var antagLoadout = LoadoutSystem.GetJobPrototype(def.PrefRoles.First());
+            var pref = (HumanoidCharacterProfile) _pref.GetPreferences(session.UserId).SelectedCharacter;
+
+            if (_prototypeManager.TryIndex(antagLoadout, out RoleLoadoutPrototype? roleProto))
+            {
+                RoleLoadout? loadout = null;
+                pref?.Loadouts.TryGetValue(antagLoadout, out loadout);
+
+                // Set to default if not present
+                if (loadout == null)
+                {
+                    loadout = new RoleLoadout(antagLoadout);
+                    loadout.SetDefault(_prototypeManager);
+                }
+
+                _stationSpawning.EquipLoadout(player, roleProto, loadout);
+            }
+
```

Comment:
This copy-paste is a bit too skrungly, at least the other one is contained to lobby code but this should be moved to a common method and used instead.

Reply:
Alright

Reply:
common method made